src/pfsys/evm/aggregation.rs

```diff
@@ -312,13 +312,13 @@ impl Circuit<Fr> for AggregationCircuit {
     }
 }
 
-/// Create aggregation EVM verifier bytecode
+/// Create aggregation EVM verifier deployment and sol code.
 pub fn gen_aggregation_evm_verifier(
     params: &ParamsKZG<Bn256>,
     vk: &VerifyingKey<G1Affine>,
     num_instance: Vec<usize>,
     accumulator_indices: Vec<(usize, usize)>,
-) -> Result<DeploymentCode, AggregationError> {
+) -> Result<(DeploymentCode, String), AggregationError> {
```

Comment:
Small thing but for clarity can we 

a) create a YulCode type which is just an alias of string `type YulCode = String` 
for legibility. 

b) we can create a gen_deployment_code function which ingests YulCode and produces DeploymentCode that way we don't have to return a tuple / it makes it clear DeploymentCode comes from YulCode. 

C) update new python.rs code accordingly.